notebooks/407-pedestrian-tracking-webcam/407-pedestrain-tracking.ipynb

```diff
@@ -0,0 +1,617 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    image1 = cv2.cvtColor(cv2.imread("data/person_1_1.png"), cv2.COLOR_BGR2RGB)
</pre><p>now, based on our agreement, all data should be stored in notebooks/data folder for avoiding data duplication</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/751/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>